azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -341,6 +352,11 @@ protected String uploadExecutableFlow(
     // The exflow id is set by the loader. So it's unavailable until after
     // this call.
     this.executorLoader.uploadExecutableFlow(exflow);
+    // Emit dispatching flow event
+    exflow.setStatus(Status.DISPATCHING);
```

Comment:
why the status is set in this PR? It should have been existing already somewhere. The event should be emitted there only.

Reply:
Removed the status